data/remnant missions.txt

```diff
@@ -493,5 +597,771 @@ mission "Remnant: Technology Available"
+			`	Almost before you are in the main hall a sales manager is at your side, inquiring as to how he can help. You can almost see the credit signs in his eyes and you suspect that scanning your license may have done a quick check of your net worth too.`
+			`	You ask a few leading questions about new designs and play the part of a captain with more money than brains, and the manager eagerly directs you to the big hangars near the end that house their biggest and fanciest ships. After it becomes clear that he doesn't have any experience flying them you politely disentangle yourselves from him to look at the ships in peace.`
+			label SkipRich
+			`	After an hour of browsing they take off to browse the outfitters and agree to meet you back at the spaceport in an hour.`
+
+mission "Remnant: Expanded Horizons Quarg 2"
+	name "Expanded Horizons Quarg Scanning"
+	description "Perform an outfit scan on the Quarg ship <npc>, then land again on <destination>."
+	source
+		planet Wayfarer
+	to offer
+		has "Remnant: Expanded Horizons Quarg 1: done"	
+	on offer
+		conversation
+			`Dawn and her fellow researchers show up a few minutes after you do, excitedly talking about the various things they have seen.`
+			`	"<first>, there is a pair of Quarg ships taking off shortly, and we were hoping you might be able to get scans of them while they are in flight. It would be really helpful for our understanding of how they work."`
+			choice
+				`	"Sure, that sounds simple enough."`
+					accept
+				`	"Sorry, that sounds risky."`
+					defer
+	
+	on accept
+		log "Volunteered to show Dawn and some of her fellow researchers the world where humans and Quarg co-exist."
+	
+	npc "scan outfits"
+		government Quarg
+		personality staying uninterested
+		fleet Quarg
+		conversation
+			`	Dawn and her compatriots eagerly pore over the readouts as they come in, then ask if you could land again to compare their scans to ones taken on the ground.`
+
+	
+	on complete
+		conversation
+			`You breath a sigh of relief once you are back on the ground without a reaction from the Quarg. You were a touch worried they might take exception to being scanned, but it appears to consider it par for the course for this system filled with human engineers.`
+			`	Dawn does some follow-up scans of idle Quarg ships, and they take some time to make sure all their pictures are well focused and accurate. They let you know that they will meet you back in the spaceport bar.`
+		payment 50000
+
+mission "Remnant: Expanded Horizons Quarg 3"
+	name "Expanded Horizons Return"
+	description "Take the Remnant researchers to <planet>"
+	passengers 3
+	blocked `This mission requires bunks for three passengers.`
+	source "Wayfarer"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Expanded Horizons Quarg 2: done"
+	on offer
+		conversation
+			`Back in the spaceport bar you wait around for a couple hours. Just when you were about to go find them, the trio walk in looking cheerful.`
```

Comment:
I was under the impression that it was more a matter of cultural or linguistic barriers to understanding them. That being said, I've revised this mission in a few ways:

Firstly, a comment early on about how this trio seem more outgoing than most of the other Remnant the player has dealt with, perhaps because they are younger, or other reasons.

Secondly, I brought back their reserve when they enter the bar. They relax and open up when they get back to the privacy of the player's ship.

Reply:
once they are in a conversation, the player can also read the excitement from their gestures, so that change of setting effectively eliminates the question about facial expressions